platform/darwin/docs/guides/For Style Authors.md.ejs

```diff
@@ -401,7 +402,7 @@ For operators that have no corresponding `NSExpression` symbol, use the
 
 ## Filtering sources
 
-You can filter a shape or vector source by setting the
+You can filter a shape or vector tile source by setting the
```

Comment:
Should this include computed shape source, or is it implied with 'shape .. source' ?

Reply:
This document is meant as a correspondence from style JSON to the runtime styling API. Since computed shape sources can’t be represented in style JSON, I’ve omitted that type here.